Signal/src/contact/OWSContactsManager.m

```diff
@@ -308,8 +309,9 @@ - (void)updateCachedDisplayNames
         }];
     });
 
-    [[NSNotificationCenter defaultCenter] postNotificationName:OWSContactsManagerSignalAccountsDidChangeNotification
-                                                        object:nil];
+    [[NSNotificationCenter defaultCenter]
+        postNotificationNameAsync:OWSContactsManagerSignalAccountsDidChangeNotification
```

Comment:
This is used to inform a bunch of components of an async event. It can be safely delivered a bit later.